%\VignetteIndexEntry{COHCAPanno Vignette}
%\VignetteKeywords{Illumina Methylation Array Annotations}
%\VignettePackage{COHCAPanno}
\documentclass{article}

\usepackage{Sweave}
\usepackage[utf8]{inputenc}
\usepackage{graphicx}
\usepackage{hyperref}
\usepackage{float}
\usepackage{cite}

\begin{document}
\title{COHCAPanno: Illumina Methylation Array Annotations for COHCAP}
\author{Charles Warden}
\maketitle

COHCAP (City of Hope CpG Island Analysis Pipeline) is an algorithm to analyze
single-nucleotide resolution methylation data (Illumina 450k methylation array,
targeted BS-Seq, etc.). It provides QC metrics, differential methylation for CpG
Sites, differential methylation for CpG Islands, integration with gene expression
data, and visualization of methylation values.

In order to make analysis easier for Bioconductor users, pre-defined annotation files
for Illumina methylation arrays are created and saved in the COHCAPanno package.  Currently,
COHCAPanno provides annotation for 27k and 450k arrays.  Detailed descriptions for each annotation
file can be found for each individual file.

Targeted BS-Seq users should create custom annotation files based upon their own data.
Likewise, users with appropriate but non-standard bisulfite-based methylation arrays can define
custom annotation files (as defined in the COHCAP package instructions).

<<loadData>>=
library("COHCAPanno")

#450 array, UCSC CpG Islands
data(COHCAP.450k.UCSC)

#450k array, HMM CpG Islands
data(COHCAP.450k.HMM)

#27k array, UCSC CpG Islands
data(COHCAP.27k)
@ 

\end{document}
